pkg/machine/wsl/machine.go

```diff
@@ -173,38 +173,40 @@ func configureSystem(mc *vmconfigs.MachineConfig, dist string, ansibleConfig *vm
 		}
 	}
 
-	lingerCmd := withUser("cat > /home/[USER]/.config/systemd/[USER]/linger-example.service", user)
-	if err := wslPipe(lingerService, dist, "sh", "-c", lingerCmd); err != nil {
-		return fmt.Errorf("could not generate linger service for guest OS: %w", err)
-	}
+	/*
```

Comment:
linger is still necessary for Podman

this can be done with

```bash
sudo systemctl enable --now systemd-logind
sudo loginctl enable-linger 1000
```